crates/starknet-devnet/tests/test_get_class.rs

```diff
@@ -279,4 +282,69 @@ mod get_class_tests {
             }
         }
     }
+
+    #[tokio::test]
+    async fn test_getting_compiled_casm_for_cairo0_or_non_existing_hash_have_to_return_class_hash_not_found_error()
```

Comment:
One of the errors that can be returned by the new RPC method is `COMPILATION_ERROR`.  It is not tested here, but it also looks improperly documented on the spec side (e.g. missing code number?). What's your opinion on this?

Reply:
compilation is being done when add_declare_transaction is invoked. Theoretically if during add_declare_transaction compilation didn't fail, so it will not fail during starknet_getCompiledCasm execution

Reply:
True. Please add a note about this, perhaps in the endpoint code.

Reply:
[1a0202b](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/633/commits/1a0202b69d4e9e4a4e27783daf907164b7870b7c)